vmdb/app/models/ems_refresh/parsers/kubernetes.rb

```diff
@@ -98,6 +98,9 @@ def parse_node(node)
 
     def parse_service(service)
       new_result = parse_base_item(service)
+      if new_result[:ems_ref] == nil # Tipically this happens for kubernetes services
```

Comment:
> @abonas correct "kubernetes" and "kubernetes-ro" seem not to have an uid anymore.

interesting :)
Maybe we should either open an issue/question on that in k8s, or perhaps ignore those entities that come without uid? they are assumed to be k8s implementation specific and not user facing, aren't they?
In any case (regardless of this patch) it is kind of inconsistent to have for same entity type sometimes a uid for ems_ref and sometimes a custom string. looks that nothing can be done with that info having several formats for it.
thoughts?